index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Great Ethiopian Run</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/main_style.css">
+    <script src="./scripts/navbar.js" defer></script>
+   <script src="./scripts/dynamic-content.js" defer></script>
+    <!-- <script src="./scripts/test-drop-down.js" defer></script> -->
+</head>
+<body>
+    <div class="home-wrapper">
+        <div class="over-nav-bar">
+            <ul class="over-nav-items">
+                <li> <a href="#"><img src="assets/icons/facebook.png" alt="facebook-icon"></a></li>
+                <li> <a href="#"><img src="assets/icons/twitter.svg" alt="twitter-icon"></a></li>
+                <li> <a href="#">English</a></li>
+            </ul>
```

Comment:
- [ ] According to the requirements and the original design, You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including Layout. So kindly, add the rest of the items in the top navbar, eg) logout, my page 👌💯

> Original Desgin
![image](https://user-images.githubusercontent.com/45231276/182980504-5e8b6d14-625a-4c66-abbd-a0269bfd6bff.png)

> Your Design
![image](https://user-images.githubusercontent.com/45231276/182980791-1cff2072-4391-46f6-ac58-ee8f5de63eb3.png)